src/plugins/home/public/application/components/_solutions_section.scss

```
@@ -1,23 +1,28 @@
 .homSolutions__item {
   @include euiBreakpoint('l', 'xl') {
-    max-width: calc(50% - #{$euiSizeM * 2});
+    max-width: calc(33.33% - #{$euiSizeM * 2});
   }
 }
 
-.euiCard__image {
-  .homSolutionPanel & {
+.homSolutionPanel {
+  .euiCard__image {
     background-color: $euiColorPrimary;
   }
 
-  .homSolutionPanel--enterpriseSearch & {
+  &--enterpriseSearch .euiCard__image {
```

Is it possible to not target the `.eui` class directly? I did a real quick test and looks like you can apply the background to the `img` tag.
```suggestion
  &--enterpriseSearch img {
```

---

Will do!